service/recognition.js

```diff
@@ -130,29 +194,106 @@ async function gratitudeErrors(gratitude) {
   ].filter((x) => x !== "");
 }
 
+async function goldenGratitudeErrors(gratitude) {
+  return [
+    !(await doesUserHoldGoldenRecognition(gratitude.giver.id, "recognizee"))
+      ? "- Only the current holder of the golden fistbump can give the golden fistbump"
+      : "",
+
+    gratitude.receivers.length === 0
+      ? "- Mention who you want to recognize with @user"
+      : "",
+    gratitude.receivers.find((x) => x.id == gratitude.giver.id)
+      ? "- You can't recognize yourself"
+      : "",
+    gratitude.giver.is_bot ? "- Bots can't give recognition" : "",
+    gratitude.giver.is_restricted ? "- Guest users can't give recognition" : "",
+    gratitude.receivers.find((x) => x.is_bot)
+      ? "- You can't give recognition to bots"
+      : "",
+    gratitude.receivers.length > 1
+      ? "- You can't give the golden fistbump to multiple users"
+      : "",
+    gratitude.receivers.find((x) => x.is_restricted)
+      ? "- You can't give recognition to guest users"
+      : "",
+    gratitude.trimmedMessage.length < minimumMessageLength
+      ? `- Your message must be at least ${minimumMessageLength} characters`
+      : "",
+    gratitude.count < 1
```

Comment:
Is it possible currently to send the golden fistbump multiple times, either by using multiple emojis or via the multiplier. If so, we may want to check the count is strictly equal to 1 here.

Reply:
Ah, perhaps this is accounted for in the feature itself, I'm seeming the count hardcoded to 1 there.